src/pages/tickets/org/orgHeader/index.tsx

```diff
@@ -458,6 +499,10 @@ export const OrgHeader = ({
     }
   }, [org_uuid, checkboxIdToSelectedMap, main, languageString]);
 
+  if (!organizationData) return null;
```

Comment:
can we try making organizationData as a required prop to avoid having to add a statement like this?

Reply:
Okay